com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/editor/controller/MigrationProgressUIController.java

```diff
@@ -450,20 +450,42 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException,
 		}
 		for (String[] item : tableItems) {
 			if (item[0].equals(tableName)) {
-				long newExp = getCellValue(item[2]) + exp;
-				item[2] = String.valueOf(newExp);
-				if (!config.isImplicitEstimate()) {
-					long oldimp = getCellValue(item[3]);
-					item[4] = String.valueOf(Math.round(100 * (newExp + oldimp)
-							/ (2 * getCellValue(item[1]))))
-							+ "%";
-				}
-				return item;
+				return getItemForExpData(exp, item);
+			}
+		}
+		return new String[] {};
+	}
+
+	public String[] updateTableExpData(String owner, String tableName, long exp) {
+		if (exp <= 0) {
+			return new String[] {};
+		}
+		for (String[] item : tableItems) {
```

Comment:
If only check item[0] and item[5] , I don't think i  need "for loop".

Reply:
* It is not the end by one time comparison, It needs to compare to columns (Table name, Table schema) in all items (Table Items) to display and update the status during UI processing. So, I think the for loop statement is required.